python/sdk/merlin/observability/inference.py

```diff
@@ -0,0 +1,111 @@
+from dataclasses import dataclass
+from enum import unique, Enum
+from typing import Dict, Optional, List
+
+from dataclasses_json import dataclass_json
+
+
+@unique
+class ValueType(Enum):
+    FLOAT64 = 1
+    INT64 = 2
+    BOOLEAN = 3
+    STRING = 4
+
+
+@dataclass_json
+@dataclass
+class RegressionOutput:
+    prediction_score_column: str
+
+    @property
+    def column_types(self) -> Dict[str, ValueType]:
+        return {self.prediction_score_column: ValueType.FLOAT64}
+
+
+@dataclass_json
+@dataclass
+class BinaryClassificationOutput:
+    prediction_label_column: str
+    prediction_score_column: Optional[str] = None
```

Comment:
can we make it mandatory and `prediction_label_column` is optional?

Reply:
Since this is merged already, i will make the changes in #488 instead.

Reply:
As for conversion to Arize schema - i am not sure if we should add dependency to Arize on the Merlin SDK level. My preference would be to keep the dependency to Arize on the consumer / flyte workflow instead.

Reply:
one drawback is easy to overlook if there is something change on the merlin part